source/extensions/common/wasm/wasm.cc

```diff
@@ -1059,9 +985,15 @@ WasmResult serializeValue(Filters::Common::Expr::CelValue value, std::string* re
     result->assign(reinterpret_cast<const char*>(&out), sizeof(absl::Time));
     return WasmResult::Ok;
   }
+  case CelValue::Type::kMessage:
+    if (value.MessageOrDie() != nullptr && value.MessageOrDie()->SerializeToString(result)) {
+      return WasmResult::Ok;
+    }
+    return WasmResult::SerializationFailure;
```

Comment:
```
auto msg = value.MessageOrDie();
if (msg == nullptr) {
    return WasmResult::SerializationFailure;
}
msg->SerializeToString(result);
return WasmResult::Ok;
```

But if `MessageOrDie()` is crashing the process (the name suggest that it does), then this could be even further simplified as simply:
```
value.MessageOrDie()->SerializeToString(result);
return WasmResult::Ok;
```

Reply:
`nullptr` is a valid value here so I changed it to treat it as Ok value with empty result.